server/utils/applicationUtils.ts

```diff
@@ -39,27 +40,36 @@ const createNameAnchorElement = (name: string, applicationId: string) => {
   return htmlValue(`<a href=${paths.applications.show({ id: applicationId })}>${name}</a>`)
 }
 
-const getResponses = (application: Application): ApplicationResponse => {
+export type ApplicationOrAssessmentResponse = Record<string, Array<PageResponse>>
+
+const getResponses = (applicationOrAssessment: Application | Assessment): ApplicationOrAssessmentResponse => {
```

Comment:
Yeah, I think this is fine for now. I do wonder if we could move some of this stuff to the form / task decorators or something. But we can handle this later 👍

Reply:
Yeah that would be smart